build.gradle

```diff
@@ -32,15 +32,16 @@ java {
 }
 
 dependencies {
-    compileOnly "org.embulk:embulk-spi:0.11"
-    implementation "org.embulk:embulk-util-config:0.5.0"
-    implementation "org.embulk:embulk-util-file:0.2.0"
-    // compile "YOUR_JAR_DEPENDENCY_GROUP:YOUR_JAR_DEPENDENCY_MODULE:YOUR_JAR_DEPENDENCY_VERSION"
-
-    testImplementation "junit:junit:4.13.2"
-    testImplementation "org.embulk:embulk-core:0.11.3"
-    testImplementation "org.embulk:embulk-deps:0.11.3"
-    testImplementation "org.embulk:embulk-junit4:0.11.3"
+    compileOnly libs.embulk.spi
+    implementation libs.embulk.util.config
+    implementation libs.embulk.util.file
+    implementation platform(libs.jackson.bom)
+    implementation libs.bundles.jackson
+
```

Comment:
This dependency doesn't contain `validation-api` and `slf4j-api`. Should I add it?

Reply:
Not mandatory because they are included as transitive dependencies, but maybe nice to have them to be consistent with other plugins.

Reply:
Thanks, I'll fix it in another PR.